routes/web.php

```diff
@@ -16,7 +17,7 @@
 */
 
 // admin routes
-Route::prefix('admin')->name('admin.')->group(function () {
+Route::prefix('admin')->middleware(IsAdmin::class)->name('admin.')->group(function () {
```

Comment:
Anh nghĩ nên dùng kiểu này nhé, chỉ cần khai báo 1 lần mà dùng nó ngắn ngon, ko phải use gì cả. 
https://laravel.com/docs/8.x/middleware#assigning-middleware-to-routes